example/spec/parallel/kinesisTests/KinesisTestTriggerSpec.js

```diff
@@ -235,7 +239,7 @@ describe('The Cloud Notification Mechanism Kinesis workflow', () => {
         await putRecordOnStream(streamName, record);
 
         console.log('Waiting for step function to start...');
-        workflowExecution = await waitForTestSf(recordIdentifier, testWorkflow, maxWaitForSFExistSecs, 'CNMToCMA');
+        workflowExecution = await waitForTestSfForRecord(recordIdentifier, workflowArn, maxWaitForSFExistSecs, 'CNMToCMA');
```

Comment:
This last argument, `CNMToCMA`, is extraneous here isn't it? `waitForTestSfForRecord` doesn't seem to accept a fourth argument?

Reply:
Correct, this is definitely some left over refactor meat.

Reply:
1432b17